src/client/client.c

```diff
@@ -326,7 +328,28 @@ int client_call_manager(Client *client) {
                         return -EINVAL;
                 }
                 r = method_lifecycle_action_on(client, client->opargv[0], client->opargv[1], "ReloadUnit");
+        } else {
+                return -EINVAL;
```

Comment:
I think you should print the usage here too. No point in failing without any explanation

Reply:
With this change, we return an error/explanation: `Call to manager failed: Invalid argument`, I figured the `Invalid argument` was enough